### PR TITLE
Changing KML file backend

### PR DESCRIPTION
**Pull request by procrastinatio**
:bomb: Do not merge until the backend is ready!

Updating to new service KML https://github.com/geoadmin/service-kml/blob/develop/app/routes.py  backend

Test:https://mf-geoadmin3.int.bgdi.ch/feature_BGDIINF_SB-2052_kml_header/2203291005/index.html

See https://jira.swisstopo.ch/browse/BGDIINF_SB-2052

Only show: https://mf-chsdi3.dev.bgdi.ch/shorten/9482e6f4e4 

Edit: https://mf-chsdi3.dev.bgdi.ch/shorten/9482e6f4ea

**PROD testing**
* Before switch: https://mf-geoadmin3.int.bgdi.ch/feature_BGDIINF_SB-2052_kml_header/2203291005/index.html?storage_url=//sys-public.prod.bgdi.ch
* After DNS switch (public.geo.admin.ch) https://mf-geoadmin3.int.bgdi.ch/feature_BGDIINF_SB-2052_kml_header/2203291005/index.html

### CODE REVIEW

File: test/specs/FileStorageService.spec.js [hunk 1]
```diff
@@ -16,14 +18,14 @@ describe('ga_filestorage_service', function() {
         fileId: fileId,
         fileUrl: 'https://public.geo.admin.ch/' + fileId
       },
-      serviceUrl, publicUrl;
+      serviceUrl;
+    form.append('kml', blob)
```
Comment (ltshb) [inline test/specs/FileStorageService.spec.js:22]: To be consistent with above would be good to also add the `author` to the form

**Review by ltshb — APPROVED** [review]
👍🏼  Just a minor comment about the `author` form field.

File: src/components/PublicStorageService.js [hunk 2]
```diff
@@ -59,18 +65,35 @@ goog.provide('ga_publicstorage_service');
...
+          } else {
+            formData.append('admin_id', adminId)
+          }
+          return $http({
+            method: method,
+            url: getServiceUrl(endPoint, fileId),
+            data: formData,
             headers: {
-              'Content-Type': contentType
+              'Content-Type': undefined
```
Comment (pakb) [inline src/components/PublicStorageService.js:89]: Is it worth keeping?

**Review by pakb — APPROVED** [review]
Small question on `Content-Type` header, otherwise looks good

File: src/components/PublicStorageService.js (hunk 2 above)
Comment (procrastinatio) [reply src/components/PublicStorageService.js:89]: Yes, it's angularjs black magic. Without it, the default `application\json` will be sent, with it, `multipart\form-data`

Comment (procrastinatio) [reply src/components/PublicStorageService.js:90]: I did comment to explain why

**Comment by procrastinatio** [comment]
Squashing code

**Pull request merged**